### PR TITLE
Formly Radio button and Search types

### PR DESCRIPTION
**Pull request by yerramshilpa**

### CODE REVIEW

File: projects/uswds-formly/src/lib/types/radio.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  template: `
+    <usa-radio-group [formControl]="formControl" [name]="to.name ? to.name : 'radio-group'"  [ariaLabelledBy]="to.ariaLabelledBy">
```
Comment (Bibek-Regmi) [inline projects/uswds-formly/src/lib/types/radio.ts:7]: Also allow `tile` option to be passed in

File: src/app/formly/radio/radio.module.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+const DEMOS = {
+    basic: {
+        title: 'Basic Radio',
+        type: FormlyBasicRadioComponent,
+        code: require('!!raw-loader!./demos/basic/radio-basic.component'),
+        markup: require('!!raw-loader!./demos/basic/radio-basic.component.html'),
+        module: require('!!raw-loader!./demos/basic/radio-basic.module'),
+        path: 'src/app/formly/radio/demos/basic',
+    },
```
Comment (Bibek-Regmi) [inline src/app/formly/radio/radio.module.ts:22]: Could you add a footer section for these formly demos documenting available template options? I.E - a user won't know that they can pass in `checked` option to pre-check a radio or `tile` to display as tile without digging through the internal code

File: projects/uswds-formly/src/lib/types/radio.ts (hunk 1 above)
Comment (yerramshilpa) [reply projects/uswds-formly/src/lib/types/radio.ts]: Updated code, will add more demo for it in a separate PR

File: src/app/formly/radio/radio.module.ts (hunk 2 above)
Comment (yerramshilpa) [reply src/app/formly/radio/radio.module.ts:22]: Sure

File: projects/uswds-formly/src/lib/types/radio.ts (hunk 1 above)
Comment (Bibek-Regmi) [reply projects/uswds-formly/src/lib/types/radio.ts]: It's OK if there isn't necessarily a demo for every template options available. As long as the user has some way to know what variations exist, that should be OK

File: projects/uswds-components/src/lib/header/header.component.spec.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
 import { Component, ElementRef } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-import { UsaHeaderModule, UsaHeaderPrimaryLink, UsaNavigationLink } from 'uswds-components';
+import { UsaHeaderModule, UsaHeaderPrimaryLink, UsaNavigationLink } from '@gsa-sam/ngx-uswds';
```
Comment (Bibek-Regmi) [inline projects/uswds-components/src/lib/header/header.component.spec.ts:4]: Thank you for updating these!

Comment (yerramshilpa) [reply projects/uswds-components/src/lib/header/header.component.spec.ts:4]: 👍

**Review by Bibek-Regmi — APPROVED** [review]

**Pull request merged**